notebooks/ir_ima_visualization/IR_IMA_Visualization_with_an_Example_of_Time_Variable_Background.ipynb

```diff
@@ -0,0 +1,923 @@
+    "    ax.plot(integ_time[1:], m[1:]*integ_time[1:], 's', markersize=25, label='Full Frame',  color='black')\n",
+    "    ax.plot(integ_time[1:], l[1:]*integ_time[1:], '<', markersize=20, label='LHS', color='C1')\n",
+    "    ax.plot(integ_time[1:], r[1:]*integ_time[1:], '>', markersize=20, label='RHS', color='C2')\n",
+    "    ax.set_ylim(0,1800)\n",
+    "    ax.grid()\n",
+    "    ax.set_xlabel('Integ. Time (s)')\n",
+    "    ax.set_ylabel('electrons')\n",
+    "    ax.legend(loc=0)\n",
+    "    _=ax.set_title(filename)\n",
+    "\n",
+    "plt.rc('xtick', labelsize=30) \n",
+    "plt.rc('ytick', labelsize=30) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "The left panel shows the observation affected by scattered light. By comparing the signal from the left side (orange triangles) and right side (green triangles) of our image, we see that the left side is brighter than the right and that the difference in total counts between the two sides is changing over the exposure. In this case, the difference between the two sides of the image is due to scattered light which varies spatially across the detector. We know the time-variable component is only due to Earth limb scattered light because this filter (F140W) is not affected by the He I 10830 Å airglow line emission (see section [7.10.2 of the WFC3 Data Handbook](https://hst-docs.stsci.edu/wfc3dhb/chapter-7-wfc3-ir-sources-of-error/7-10-time-variable-background)). In the following sections, we will identify the impacted reads by looking at the difference in count rates between each read.\n",
+    "\n",
+    "The right panel shows the ramp plot for the nominal exposure, and we see that the signal from the left side, right side, and full frame are nearly identical. \n",
+    "\n",
+    "**Note that, in an image with an extended object, there may be a spatial dependence in the ramp plots (e.g. the right side may be higher than the left), but the slope of the ramp would be linear with time.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Taking the Difference Between IMA Reads: The Cumulative Rate <a id=\"cumulative\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Our first method of taking the difference between reads is as simple as subtracting each read from the one before,        e.g. $\\mu$ = [SCI,15]-[SCI,14], [SCI,14]-[SCI,13],... where $\\mu$ represents the mean difference.  In the panels below, we now plot the the ramps in units of count rate (e-/s) in order to see how the background changes through the exposure. We use the *cumulative_diff* function from the *DifferenceMethods.py* module, which computes the difference between reads as described above, and the *plotramp* function from the module, which plots the median difference in count rate from the left side, right side, and full frame image.\n",
```

Comment:
is simply subtracting each read from the one before.

Also aren't you finding the median difference?